index.html

```diff
@@ -65,6 +65,67 @@ <h3>Use case #2: Efficiency enhancements during RTC</h3>
         </p>
       </section>
     </section>
+    <section id="examples">
+      <h2>Examples</h2>
+      <p>The <a data-cite="screen-capture/#example">basic example from
+        the Screen Capture specification</a> is extended below by passing
+        to <code>getDisplayMedia()</code> a <code>CaptureController</code>
```

Comment:
I think this line is distracting. I don't remember who authored that original example, but given how brief it is, I don't think they'd care much about credit (which referencing the spec doesn't really give them anyway).

Reply:
OK, the idea was to rely on an existing known example so people only need to think about the minor differences, but yes probably not useful.